en/_posts/2016-04-20-quick-start-with-marathon.md

```diff
@@ -6,16 +6,16 @@ author: devatoria
 lang: en
 ---
 
-Welcome to the quick start tutorial of the PaaS Docker lab. This document will guide you to launch your first application and play with it.
+Welcome to the quick start tutorial of the PaaS Docker lab. This document will help you launch your first application using the Marathon Web UI.
 
-#Access to your interface
+#Access to your Web UI
 
-After requesting your stack deployment through RunAbove labs section, you can access your Marathon web access using the given URL.
+Once your Mesos slave has been installed, you'll receive an email containing the URL and credentials of your marathon.
 
 
 ![Main interface](/kb/images/2016-04-20-quick-start-with-marathon/marathon.png)
- 
-Here you can see a lot of things you can do. Let's look around:
+
+As you can see, there's a lof of things you can do. Let's have a look:
```

Comment:
(non blocking suggestion) "This screenshot gives a first taste of what can be done with Marathon:"